install/kubernetes/tetragon/templates/operator_clusterrole.yaml

```
@@ -26,12 +26,14 @@ rules:
       - patch
       - update
       - watch
+  {{- if not .Values.tetragonOperator.skipCRDCreation }}
```

```suggestion
  {{- if eq .Values.crds.installMethod "operator" }}
```
It needs to be changed as `tetragonOperator.skipCRDCreation` value got deprecated & removed in the meantime.

---

Okay I've done the changes as per that.

---

Hey @itsCheithanya I don't see new changes, did you forget to push maybe?

---

looks good now, thanks!